tests/test_workers/test_cleanup_job.py

```diff
@@ -0,0 +1,71 @@
+from unittest import mock
+from datetime import datetime
+
+from cachito.workers.cleanup_job import main
+
+
+expected = {
+    "items": [
```

Comment:
Could you use single quotes to be consistent with the rest of the project? It's annoying to go back and forth.

Reply:
doh! I had it in mind while writing the tests and then it slipped. I apologize. Changing it now...